lib/minesweeper_2pl/board.rb

```diff
@@ -1,4 +1,31 @@
 module Minesweeper_2pl
+
   class Board
+    attr_accessor :size, :bomb_count, :bomb_positions, :positions, :row_size
+
+    def set_positions(positions)
+      set_row_size(positions)
+      set_bombs
+      @positions = []
+      positions.times do |position|
+        if self.bomb_positions.include?(position)
```

Comment:
Take a look at [`Array#map`](https://ruby-doc.org/core-2.2.0/Array.html#method-i-map) in Ruby. There may be a way to do this transformation more concisely.